sdk/Sdk.Analyzers/DiagnosticDescriptors.cs

```diff
@@ -16,13 +13,17 @@ private static DiagnosticDescriptor Create(string id, string title,string messag
             return new DiagnosticDescriptor(id, title, messageFormat, category, severity, isEnabledByDefault: true, helpLinkUri: helpLink);
         }
 
-        public static DiagnosticDescriptor WebJobsAttributesAreNotSuppoted { get; }
+        public static DiagnosticDescriptor WebJobsAttributesAreNotSupported { get; }
             = Create(id: "AZFW0001", title: "Invalid binding attributes", messageFormat: "The attribute '{0}' is a WebJobs attribute and not supported in the .NET Worker (Isolated Process).",
                 category: Constants.DiagnosticsCategories.Usage, severity: DiagnosticSeverity.Error);
-                
+
         public static DiagnosticDescriptor AsyncVoidReturnType { get; }
             = Create(id: "AZFW0002", title: "Avoid async void methods", messageFormat: "Do not use void as the return type for async methods. Use Task instead.",
                 category: Constants.DiagnosticsCategories.Usage, severity: DiagnosticSeverity.Error);
 
+        public static DiagnosticDescriptor DeferredBindingAttributeNotSupported{ get; }
+            = Create(id: "AZFW0003", title: "Invalid class attribute", messageFormat: "The attribute '{0}' can only be used on trigger and input binding attributes.",
```

Comment:
@liliankasem AZFW0003 is an [existing one](https://github.com/Azure/azure-functions-dotnet-worker/blob/2b78e7b03503f499ae71507b2a842d1848b86fb2/sdk/Sdk.Generators/DiagnosticDescriptors.cs#L18). So we should use AZFW0009 or above.
Also we need to update the docs [here](https://github.com/Azure/azure-functions-dotnet-worker/tree/main/docs/analyzer-rules).

Reply:
Will PR to fix!